xgcm/grid.py

```diff
@@ -45,7 +45,8 @@ class Axis:
     differentiated by their length.
     """
 
-    def __init__(self, ds, axis_name, periodic=True, default_shifts={}):
+    def __init__(self, ds, axis_name, periodic=True, default_shifts={},
+                 wrap=None):
```

Comment:
the `wrap` option needs to be added to the docstring for the class.

What does `wrap` do? How does it relate to the `periodic` option?